.gitignore

```diff
@@ -1,2 +1,2 @@
-middleclickclose@paolo.tranquilli.gmail.com.zip
-gschemas.compiled
```

Comment:
I think it's alright for the pot file to be committed, whereas gschemas.compiled still needs to be ignored (for development purposes it's easier to run the extension directly from the git repo)

Reply:
The way I did development was to just run `make install` after every change and then restart gnome-shell and it worked great (except the fact that I had to run a nested session because gnome 45 beta-1 seems to crash on restart for some reason, but anyway).

I didn't commit the pot file, because I couldn't find a way to make `make pot` idempotent (the creation timestamp would update every time). Maybe it's worth keeping a copy of the pot file `make po` was run with somehow? But I'm not sure how I would do that while keeping `make pot` idempotent.